### PR TITLE
Add govuk-frontend-jinja and digitalmarketplace-govuk-frontend

### PR DESCRIPTION
**Pull request by risicle**
https://trello.com/c/Jwpu5Zg2

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -236,6 +239,24 @@ gulp.task(
   )
 )
 
+gulp.task(
+  'copy:govuk_frontend_assets:fonts',
+  copyFactory(
+    'fonts from the GOVUK frontend assets',
+    govukFrontendFontsFolder,
+    assetsFolder + '/fonts'
```
Comment (lfdebrux) [inline gulpfile.js]: This should actually be `staticFolder`; see https://github.com/alphagov/digitalmarketplace-user-frontend/pull/176

Comment (risicle) [reply gulpfile.js]: ah ok

File: gulpfile.js [hunk 2]
```diff
@@ -286,9 +309,9 @@ gulp.task('build:development', gulp.series(gulp.parallel('set_environment_to_dev
 gulp.task('build:production', gulp.series(gulp.parallel('set_environment_to_production', 'clean'), 'compile'))
 
 gulp.task('watch', gulp.series('build:development', function () {
-  const jsWatcher = gulp.watch([path.join(assetsFolder, '**', '*.js')], gulp.series('js'))
-  const cssWatcher = gulp.watch([path.join(assetsFolder, '**', '*.scss')], gulp.series('sass'))
-  const dmWatcher = gulp.watch([path.join(npmRoot, 'digitalmarketplace-frameworks', '**')], gulp.series('copy:frameworks'))
+  const jsWatcher = gulp.watch([assetsFolder + '/**/*.js'], gulp.series('js'))
+  const cssWatcher = gulp.watch([assetsFolder + '/**/*.scss'], gulp.series('sass'))
+  const dmWatcher = gulp.watch([npmRoot + '/digitalmarketplace-frameworks/**'], gulp.series('copy:frameworks'))
```
Comment (aliuk2012) [inline gulpfile.js]: Why have we stopped using `path.join`?

Comment (risicle) [reply gulpfile.js]: I must have copypasta'd it from an example that doesn't... and sure enough the user-frontend doesn't either in these lines.

Comment (aliuk2012) [reply gulpfile.js]: User-frontend should use path.join. I'll do a pr for that. could you revert these cahnges?

**Pull request merged**